CardView: describe scrolling and dragging API (#7491)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxCardView/9 Types/HeaderPanel/dragging/dropFeedbackMode.md

---
id: HeaderPanel.dragging.dropFeedbackMode
type: Enums.DragHighlight
default: 'push'
---
---
##### shortDescription
Specifies how to highlight the header item's drop position.

---
This property accepts the following values:

- *"indicate"*    

>   Header items do not animate, a blue line appears at the place where the dragged item is going to be placed.

Reviewer: ```suggestion
  Header items are not animated; a blue line appears where the dragged item is going to be placed.
```

- *"push"*    

>   Header items move apart with animation, allowing room for the dragged row to be placed.

Reviewer: ```suggestion
  Header items move apart with an animation, allowing room for the dragged row.
```
